cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -274,6 +274,20 @@
 
     {% block javascript scoped %}
 
+    {# TODO: Remove when 2019/2020 Mega Menu modifications are finalized. #}
+    {% if flag_enabled('MEGA_MENU_VAR_1', request) %}
+      <script>
+        window.cfpb = window.cfpb || {};
```

Comment:
Is this an existing technique (using `window.cfpb` to stash configuration variables), or a new approach?

I wonder (ping @willbarton) how hard it would be to generate a list of enabled flags, and stash that in the DOM. So we'd have another [django-flags](https://github.com/cfpb/django-flags) call like `get_enabled_flags()` and then we could do something like:

```html
<script>
  if (!window.cfpb) {
    window.cfpb = {
      flags: {{ get_enabled_flags() | json }}
    }
  }

   // Then to test a flag you could do something like:
  if (window.cfpb.flags.MEGA_MENU_VAR_1) {
    ...
  }
</script>
```

If we made this part of the base template then FEWDs could more easily generally test for flags without having to hardcode each one in the template.

One downside might be that it might be inefficient to test flag state for all flags at once.

Reply:
> Is this an existing technique (using window.cfpb to stash configuration variables), or a new approach?

New approach. I thought I should namespace it for safety.